dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/ZonedIntervalIterable.java

```diff
@@ -108,9 +134,13 @@ public static ZonedIntervalIterable forConnectorArguments(@Nonnull ConnectorArgu
           "At least one day of query logs should be exported; you specified: " + daysToExport);
 
     LOG.info(
-        "Log entries within the last {} days will be exported in increments of 1 hour.",
-        daysToExport);
-    return ZonedIntervalIterable.forTimeUnitsUntilNow(24 * daysToExport, ChronoUnit.HOURS);
+        "Log entries within the last {} days will be exported in increments of 1 {}.",
+        daysToExport,
+        timeUnit);
+
+    int multiplicationFactor = timeUnit.equals(ChronoUnit.HOURS) ? 24 : 1;
```

Comment:
This will break if we allow other units. Might be better to be explicit and fail if the unit is neither `HOURS` nor `DAYS`.

Reply:
Agreed. Applied